plugins/kotlin-as-java/src/main/kotlin/KotlinAsJavaPageContentBuilder.kt

```diff
@@ -0,0 +1,185 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.Function
+import org.jetbrains.dokka.model.TypeWrapper
+import org.jetbrains.dokka.model.doc.DocTag
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.transformers.psi.JavaTypeWrapper
+import org.jetbrains.dokka.utilities.DokkaLogger
+
+class KotlinAsJavaPageContentBuilder(
+    private val dri: Set<DRI>,
+    private val platformData: Set<PlatformData>,
+    private val kind: Kind,
+    private val commentsConverter: CommentsToContentConverter,
+    val logger: DokkaLogger,
+    private val styles: Set<Style> = emptySet(),
+    private val extras: Set<Extra> = emptySet()
+) : PageContentBuilder {
```

Comment:
Subclassing `DefaultPageContentBuilder` is a good idea there IMHO, as some methods are just duplicates